bin/segment2.c

```diff
@@ -16,29 +16,56 @@ ALIGNED8 static const Texture small_font_1[] = {
 ALIGNED8 static const Texture small_font_2[] = {
 #include "textures/segment2/custom_text2.i4.inc.c"
 };
+ALIGNED8 static const Texture small_font_3[] = {
+#include "textures/segment2/custom_text3.i4.inc.c"
+};
+ALIGNED8 static const Texture small_font_4[] = {
+#include "textures/segment2/custom_text4.i4.inc.c"
+};
+
+
+const Texture *const puppyprint_font_lut[] = {
+    small_font_1, small_font_2, small_font_3, small_font_4
+};
```

Comment:
i dont see why these should always be included in the rom with `PUPPYPRINT`. i would prefer to have individual configuration here, like having `enum PuppyFont` control which fonts should be included in the rom.